src/status_im/ethereum/json_rpc.cljs

```diff
@@ -32,6 +32,8 @@
    "shhext_loadFilters" {}
    "shhext_loadFilter" {}
    "shhext_removeFilters" {}
+   "shhext_chats" {}
```

Comment:
what was the difference between shhext_chats and status_chats?

Reply:
`status_chats` was just a stub endpoint, no logic, just hardcoded chats